docs/architecture/adr-064-batch-verification.md

```diff
@@ -0,0 +1,85 @@
+# ADR 064: Batch Verification
+
+## Changelog
+
+- January 28, 2021: Created (@marbar3778)
+
+## Context
+
+Tendermint uses public private key cryptography for validator signing. When a block is proposed and voted on validators sign a message representing acceptance or rejection of a block. These signatures are also used to verify previous blocks are correct if a node is syncing. Currently, Tendermint requires each signature to be verified individually, this leads to a slow down of block times.
+
+## Alternative Approaches
+
+- Signature aggregation
+  - Signature aggregation is an alternative to batch verification. Signature aggregation leads to fast verification and smaller block sizes. At the time of writing this ADR there is on going work to enable signature aggregation in Tendermint. The reason why we have opted to not introduce it at this time is because every validator signs a unique message. One of the two benefits can be done, aggregation, but verification would suffer because the software would need to verify each signature individually due to their uniqueness. For example if we were to implement signature aggregation with BLS, there could be a potential slow down of 10x-100x in verification speeds.
```

Comment:
It would be helpful to first define or describe what batch verification is first before stating the alternative and why one is preferable over the other (in the context of Tendermint) IMHO.

This could perhaps be done in a sentence or two in the `Context`.

Reply:
> At the time of writing this ADR there is on going work to enable signature aggregation in Tendermint.

Maybe site these in footnotes?

Reply:
> One of the two benefits can be done, aggregation, but verification would suffer because the software would need to verify each signature individually due to their uniqueness.

This sentence reads a bit weird. I don't understand it.

Reply:
removed the sentence. Added a few on batch verification and references to various issues and specs